src/backends/tensorrt/plan_utils.cc

```diff
@@ -348,6 +348,39 @@ ValidateDimension(
   return Status::Success;
 }
 
+Status
```

Comment:
Can we have a template for the two ValidateDimension function? As the code looks identical.

Reply:
Done